lib/model/model.go

```diff
@@ -1949,7 +1949,7 @@ func (m *Model) generateClusterConfig(device protocol.DeviceID) protocol.Cluster
 		protocolFolder := protocol.Folder{
 			ID:                 folder,
 			Label:              folderCfg.Label,
-			ReadOnly:           folderCfg.Type == config.FolderTypeReadOnly,
+			ReadOnly:           folderCfg.Type == config.FolderTypeSendOnly,
```

Comment:
I guess so, yes. However that does not need to be part of *this* PR, as it does not introduce another folder type.